include/ocpp/common/charging_station_base.hpp

```diff
@@ -7,6 +7,8 @@
 #include <ocpp/common/pki_handler.hpp>
 #include <ocpp/common/websocket/websocket.hpp>
 
+#include <boost/shared_ptr.hpp>
```

Comment:
Why is the boost shared_ptr still used?

Reply:
Because otherwise libocpp does fail to build.